lib/widgets/atoms/explore_filter.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import '../../constants/app_constants.dart';
+
+class ExploreFilter extends StatelessWidget {
+  final UserType userType;
+  final bool userFiltersSelected;
+  final List<String?>? skills;
+  final List<String?>? languages;
+  final List<String?>? countries;
```

Comment:
nit: You could assume that if any of the filter lists is not empty then userFiltersSelected will be true, having to set both the filters and the flag is a bit redundant. You could have the flag as a private instance field that is initialized based on whether any of the lists (skills, languages, countries) is not empty. 

Not a huge deal though, just a small optimization :-)